subprojects/diagnostics/src/integTest/groovy/org/gradle/api/tasks/diagnostics/HelpTaskIntegrationTest.groovy

```diff
@@ -45,6 +45,8 @@ To see a list of command-line options, run gradle --help
 
 To see more detail about a task, run gradle help --task <task>
 
+For troubleshooting, visit https://help.gradle.org
```

Comment:
It's kind of weird that we don't use dots at the end of any sentence of the help task output.